packages/components/src/date-time/style.scss

```diff
@@ -54,19 +54,35 @@
 		font-size: $default-font-size;
 	}
 
+	// Seperate borders so that border respect border radius
+	.CalendarMonth_table {
+		border-collapse: separate;
+		border-spacing: 2px;
+	}
+
 	.CalendarDay {
 		font-size: $default-font-size;
-		border: 1px solid transparent;
+		border: none;
```

Comment:
By setting this to none, no border should show up on regular days in Windows 10 high contrast mode.